nni/algorithms/compression/v2/pytorch/pruning/tools/base.py

```diff
@@ -375,10 +375,15 @@ def _expand_mask(self, name: str, mask: Tensor) -> Dict[str, Tensor]:
         weight_mask = mask.clone()
 
         if self.block_sparse_size is not None:
+            block_sparse_size = list(self.block_sparse_size)
```

Comment:
you have already made `self.block_sparse_size` be a list, why use `list()` again?